app/jobs/reports/idv_legacy_conversion_supplement_report.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module Reports
+  class IdvLegacyConversionSupplementReport < BaseReport
+    REPORT_NAME = 'idv-legacy-conversion-supplement-report'
+
+    def perform(_date)
+      csv = build_csv
+      save_report(REPORT_NAME, csv, extension: 'csv')
+    end
+
+    # @return [String] CSV report
+    def build_csv
+      sql = <<~SQL
+        SELECT iaa_orders.*,
+                iaa_gtcs.gtc_number AS gtc_number,
+                upgrade.issuer AS issuer,
+                sp.friendly_name AS friendly_name,
+                DATE_TRUNC('month', upgrade.upgraded_at) AS year_month,
+                count(distinct upgrade.user_id) AS user_count
```

Comment:
let's move to leading commas and avoid the `iaa_order.*` if we could too, per [our SQL style guide](https://handbook.login.gov/articles/sql-style-guide.html)


```suggestion
        SELECT
          iaa_orders.start_date
        , iaa_orders.end_date
        , iaa_orders.iaa_order_number
        , iaa_gtcs.gtc_number AS gtc_number
        , upgrade.issuer AS issuer
        , sp.friendly_name AS friendly_name
        , DATE_TRUNC('month', upgrade.upgraded_at) AS year_month
        , COUNT(DISTINCT upgrade.user_id) AS user_count
```

Reply:
Sweet! Thanks for the link. Adjusted to match (order_number)